docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -3975,3 +3976,22 @@ arrow_typeof(expression)
 +---------------------------+------------------------+
 1 row in set. Query took 0.001 seconds.
 ```
+
+### `version`
+
+Returns the version of DataFusion.
+
+```
+version()
+```
+
+#### Example
+
+```
+> select version();
++--------------------------------------------+
+| version()                                  |
++--------------------------------------------+
+| Apache DataFusion 41.0.0, aarch64 on macos |
```

Comment:
note! this will go out of date very quickly, do we care?

Reply:
I don't think it matters if the doc example doesn't reflect the current version